examples/cloudml-document-ai-patents/automl_ner.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Runs AutoML NER on the text and writes results to BigQuery."""
+
+import logging
+import os
+import utils
+
+from google.cloud import storage, bigquery
+from google.cloud import automl_v1beta1 as automl
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def extract_field_from_payload(text, payload, field_name, default_value='None'):
+  """Parses a payload to extract the value of the given field.
+
+  Args
+    text: text analyzed by AutoML NER.
+    payload: payload returned by AutoML NER.
+    field_name: Name of the field to extract.
+    default_value: Value to return if the field can not be found.
+
+  Returns:
+    extracted value.
+
+  In case the payload contains several times the given field, we take the occurence
+    with the highest score.
```

Comment:
Maybe worth renaimng the method to get_highest_scoring_string or something like that?